illumina_pipeline.pl

```diff
@@ -270,6 +270,7 @@ sub checkConfig {
     elsif(! -e $opt{GENOME}){ print"ERROR: $opt{GENOME} does Not exist\n"}
     if(! $opt{SAMBAMBA_PATH}){ print "ERROR: No SAMBAMBA_PATH option found in config files.\n"; $checkFailed = 1; }
     if(! $opt{QUEUE_PATH}){ print "ERROR: No QUEUE_PATH option found in config files.\n"; $checkFailed = 1; }
+    if(! $opt{SET_HAS_METADATA}){ print "ERROR: No SET_HAS_METADATA option found in config files.\n"; $checkFailed = 1; }
```

Comment:
Sure, this will need to be added to any new INI files. We don't use the ini files inside the repo here (kind of the point of the metadata, move configuration and parameters outside of the functionality of the pipeline), but I'm happy to add it to the others too.